tests/lib/smf/src/test_lib_flat_smf.c

```diff
@@ -36,8 +36,8 @@
 #define STATE_C_EXIT_BIT        (1 << 8)
 
 #define TEST_ENTRY_VALUE_NUM     0
-#define TEST_RUN_VALUE_NUM       4
-#define TEST_EXIT_VALUE_NUM      8
+#define TEST_RUN_VALUE_NUM       1
```

Comment:
Note - SMF_RUN define at line 24 is no longer used.

Reply:
replaced with SMF_NUM_STATES, which is used in transition test